contrib/cirrus/lib.sh

```diff
@@ -250,9 +248,11 @@ use_cni() {
 use_netavark() {
     req_env_vars OS_RELEASE_ID PRIOR_FEDORA_NAME DISTRO_NV
     local magickind repokind
-    msg "Forcing NETWORK_BACKEND=netavark for all subsequent environments."
-    echo "NETWORK_BACKEND=netavark" >> /etc/ci_environment
-    export NETWORK_BACKEND=netavark  # needed for install_test_configs()
+    msg "Unsetting NETWORK_BACKEND for all subsequent environments."
+    echo "export -n NETWORK_BACKEND" >> /etc/ci_environment
+    echo "unset NETWORK_BACKEND" >> /etc/ci_environment
+    export -n NETWORK_BACKEND
+    unset NETWORK_BACKEND
```

Comment:
I've never understood why this was handled via `unset`: I like explicit settings, not assumptions of knowing what the default will be. Unless there's a need to repush, though, this  is fine.